react/features/base/util/configUtil.js

```diff
@@ -0,0 +1,85 @@
+/* @flow */
+
+declare var config: Object;
+
+/**
+ * Defines some utility methods that are used before the other JS files are
+ * loaded.
+ */
+
+
+/**
+ * Builds and returns the room name.
+ *
+ * @returns {string}
+ */
+export function getRoomName(): string | typeof undefined {
```

Comment:
I think config related stuff should go into some features/base/config feature. @lyubomir WDYT?